endoreg_db/utils/storage.py

```
@@ -77,7 +77,7 @@ def delete_field_file(
     if not _has_field_file(field_file):
         return False
     try:
-        field_file.delete(save=save)  # type: ignore[union-attr]
+        field_file.delete(save=save)[union - attr]
```

Pylint (4.0.2)

[error] Undefined variable 'union'

(E0602)